src/openrct2/rct2/S6Importer.cpp

```diff
@@ -919,7 +919,7 @@ class S6Importer final : public IParkImporter
     void ImportResearchList()
     {
         bool invented = true;
-        for (size_t i = 0; i < sizeof(_s6.research_items); i++)
+        for (size_t i = 0; i < std::size(_s6.research_items); i++)
```

Comment:
Since the size is known at compile-time and the index is not used for anything else, this could be refactored to a for-range loop.

```suggestion
        for (RCT12ResearchItem& ri : research_items)
```

Reply:
This doesn't compile. Should have been `_s6.research_items`.